addons/spreadsheet/static/src/list/plugins/list_core_plugin.js

```diff
@@ -271,6 +191,11 @@ export default class ListCorePlugin extends CorePlugin {
         };
     }
 
+    //TODO find better name
+    getListFullDefinition(id) {
```

Comment:
Name kinda sucks (same for pivots).

Getter is needed because `getListDefinition` unpack the definition to be pretty and human usable. I guess I could use the standard getter and re-pack the definion into an `{metadata, searchParams}` object but that feels stupid.

`getListRawDefinition` ? `getListDefinitionForDatasource` (but then there's mention of datasource in the core plugin)... Idk

Reply:
`getPivotModelDefinition` it is